bedrock/firefox/templates/firefox/browsers/compare/index.html

```diff
@@ -19,26 +21,21 @@
 {% endblock %}
 
 {% block content %}
-<section class="mzp-c-hero mzp-has-image compare-hero">
-  <div class="mzp-l-content">
-    <div class="mzp-c-hero-body">
-
-      <p class="compare-hero-subtitle">{{ ftl('compare-index-privacy-utility-portability') }}</p>
-
-      <h1 class="mzp-c-hero-title">{{ self.page_title() }}</h1>
-
-      <div class="mzp-c-hero-desc">
-        <p>{{ ftl('compare-index-looking-for-a-better-browser', fallback='compare-index-looking-for-a-better-fallback') }}</p>
-      </div>
-
-      {{ download_firefox(dom_id='compare-hero-download', download_location='primary cta') }}
-
+{% call split(
+    image_url='img/firefox/compare/compare-browser-windows.png',
+    include_highres_image=True,
+    block_class='',
+    theme_class='',
```

Comment:
Do these need to be defined as empty strings? (the defaults seem to be `None` in the macro)

Reply:
Oops, that was a leftover from pasting that I neglected to remove.